config/v1/types_infrastructure.go

```diff
@@ -648,6 +648,18 @@ type GCPPlatformStatus struct {
 	// +optional
 	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
 	ClusterHostedDNS ClusterHostedDNS `json:"clusterHostedDNS,omitempty"`
+
+	// cloudLoadBalancerIPs contains the IP addresses of API, API-Int and
+	// Ingress Load Balancers created on the cloud platform. Its values would
+	// not be populated for on-prem platforms. These Load Balancer IPs are
+	// used to configure the in-cluster DNS instance for API, API-Int and
+	// Ingress services. `clusterHostedDNS` is expected to be `Enabled` when
+	// these Load Balancers are populated and used.
```

Comment:
Can they be used without the clusterHostedDNS?

Reply:
No. So, the implementation in the Installer will check for that